src/Sulu/Bundle/CustomUrlBundle/Controller/CustomUrlController.php

```diff
@@ -0,0 +1,149 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\CustomUrlBundle\Controller;
+
+use FOS\RestBundle\Controller\Annotations\RouteResource;
+use Hateoas\Representation\CollectionRepresentation;
+use Hateoas\Representation\RouteAwareRepresentation;
+use Sulu\Component\Rest\RequestParametersTrait;
+use Sulu\Component\Rest\RestController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Provides rest api for custom-urls.
+ *
+ * @RouteResource("custom-urls")
+ */
+class CustomUrlController extends RestController
+{
+    use RequestParametersTrait;
+
+    private static $relationName = 'custom-urls';
+
+    /**
+     * Returns a list of custom-urls.
+     *
+     * @param string $webspaceKey
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function cgetAction($webspaceKey, Request $request)
+    {
+        // TODO pagination
```

Comment:
Will the response change if we add that later? Otherwise it doesn't matter to me.

But isn't a datagrid used here? Without any pagination? Would it really be so much work to implement that now?